src/controller/AutoCommissioner.h

```diff
@@ -25,6 +25,14 @@ namespace Controller {
 
 class DeviceCommissioner;
 
+// It seems like this should come from the zap
```

Comment:
It would if the XML were not buggy... Created https://github.com/project-chip/connectedhomeip/pull/13845 but using those here can be a followup.

Reply:
Ack - thanks. This was not working when I wrote this. Will update in a followup

Reply:
Fixed in a the next commit. Thanks for adding that.